src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -96,6 +97,7 @@ struct sd_dhcp_client {
         size_t client_id_len;
         char *hostname;
         char *vendor_class_identifier;
+        char *user_class;
```

Comment:
this is not the right type. we should store things internally always with their native types. And given that this takes a list of strings, we should really make this an strv, i.e. a char**

Reply:
Please see my review. I think there's no reason why this must be a string. The RFC only talks about a sequence of octets with a length prefix. I requested for this to be changed to an iovec.

Reply:
updated.